### ISSUE TITLE
Problem on WAMP with Directory separator

### ISSUE THREAD

**Issue by Jellygnite** [silverstripe/silverstripe-docsviewer#70]
Children menu in sidebar is not displaying due to DIRECTORY_SEPARATOR in  function getChildrenFor in DocumentManifest.php (line 621)

```
        // only pull it up if it's one more level depth
        if(substr_count($pagePath,DIRECTORY_SEPARATOR ) == ($depth + 1)) {
```

should be

```
        // only pull it up if it's one more level depth
        if(substr_count($pagePath, '/') == ($depth + 1)) {
```

**Comment by camfindlay** [comment]
DIRECTORY_SEPARATOR looks like a php constant that you can set... this would also for different separators perhaps?

**Comment by dhensby** [comment]
You can always use / on windows because the OS is not picky about the separator.

**Comment by tractorcow** [comment]
yes but substr_count isn't agnostic on windows.

**Comment by dhensby** [comment]
Sorry, yes, I'm agreeing with this suggestion but agreeing with normalising to fixed `/`

> What could also be done in this case is to normalise all separators (either to a fixed / or DIRECTORY_SEPARATOR) before doing substr_count, although the suggestion above is probably fine. :)

**Comment by tractorcow** [comment]
:D

**Comment by robbieaverill** [comment]
Fixed in #97
